docs/search.html

```diff
@@ -8,7 +8,7 @@
   
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
-  <title>Search &mdash; integreat-cms 0.0.1 documentation</title>
```

Comment:
is it possible to use a parameter for the version number?

Or isd this version autogenerated?

Reply:
This is autogenerated and derived from the [sphinx config file](https://github.com/Integreat/cms-django/pull/356/files#diff-5820bd443376e6f878b54152642ea8aaR40). However, I thought this represents the version of the documentation, but it should correspond to the cms version number in [setup.py](https://github.com/Integreat/cms-django/blob/85d53739e9d1f9434bc65870bcb4283f21b1b076/setup.py#L9), right? Then I'd have to find a way to take the package version number here...